test/module.js

```diff
@@ -0,0 +1,134 @@
+var tests = {};
```

Comment:
I didn't see that file. At first, I thought that given the fact that the new methods are all related to `QUnit.module`, I will put them all in one file and the only name I had at a time was `module`.

I will update it to reflect the implemented method.
Thanks!